README.md

```diff
@@ -43,6 +43,10 @@ meson setup build
 meson compile -C build
 ```
 
+Once Spiel is built, to run test the Python example above or a similar GObject client, make sure you are inside the build environment by running `meson devenv -C build`.
+
+To install libspiel system wide and expose it to GObject client code without needing to run `meson devenv`, run $TODO_NOT_SURE
```

Comment:
```suggestion
To install libspiel system wide and expose it to GObject client code without needing to run `meson devenv`, run 
```
Need to determine what to fill in here.

Reply:
`meson install -C build`?

Reply:
Thanks! Added that to this PR. Sorry if this is trivial. I have not used Meson in the past. This is all set on my end if you think it is reasonable to document.